docs/myrocks/variables.md

```diff
@@ -758,6 +777,25 @@ When enabled this variable allows/encourages threads that are using
 two-phase commit to `prepare` in parallel. This variable was
 renamed in upstream to rocksdb_two_write_queues.
 
+### `rocksdb_converter_record_cached_length`
+
+| Option       | Description                  |
+|--------------|------------------------------|
+| Command-line | --rocksdb_converter_record_cached_length |
+| Dynamic      | Yes                           |
+| Scope        | Global                        |
+| Data type    | Numeric                       |
+| Default      | 0                             |
+
+The variable was implemented in [Percona Server for MySQL 8.0.33-25](../release-notes/Percona-Server-8.0.33-25.md).
+
+Specifies the maximum number of bytes to cache on table handler for encoding table record data. 
```

Comment:
Please add:
`If the used memory exceeds rocksdb_converter_record_cached_length, the memory is released when the handler is returned to table handler cache.`

Reply:
Added